spec/order_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'Takeout'
+
+describe Takeout do
+  let(:Pizza) {double :Pizza => 6}
+  let(:Water) {double :Water => 1}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space missing inside {.<br>Space missing inside }.